modules/mig/README.md

```diff
@@ -5,7 +5,7 @@ and optionally, an autoscaler and healthchecks.
 
 ## Usage
 
-See the [simple example](../../examples/mig/simple) for a usage example.
+See the [simple example](https://github.com/terraform-google-modules/terraform-google-vm/tree/master/examples/mig/simple) for a usage example.
```

Comment:
This will need to modified in https://github.com/terraform-google-modules/terraform-google-vm/blob/c95056c5236d7faba1497e1eb54e3938a0c35ba1/autogen/README.md?plain=1#L15